Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md

```diff
@@ -6,53 +6,103 @@
 ### Required Policies
 Control can be covered with Azure policies mentioned below:
 - Policy to enable Microsoft Defender with Standard tier for SQL servers at Subscription level (if not enabled).
-- Policy to configure email address in Microsoft Defender security contacts (if security contacts not setup as per requirements).
-- Policy to enable Advanced Threat Protection (ATP) for each non-compliant SQL server.
-- Policy to enable SQL auditing for each non-compliant SQL server.
-
+- Policy to configure/audit email address in Microsoft Defender security contacts (if security contacts not setup as per requirements).
+- Policy to audit/enforce Advanced Threat Protection (ATP) for each non-compliant SQL server.
+- Policy to audit/enforce SQL auditing for each non-compliant SQL server.
+- Policy to audit Security Alert contacts for SQL server (if security contacts not setup as per requirements).
 ___ 
 
 #### Policy Details
 
-Following policy will enable Microsoft Defender with Standard tier for SQL servers at Subscription scope. This will help in creation of protected resources.
+Following policy can be used to audit/enforce SQL Auditing on SQL server.
+ > **Important**: 
+    1. Two different policy definitions are required to cover both general SQL servers and SQL servers which are part of Synapse Workspace as policy aliases are different.  
+    2. In the provided resource group, Storage account will be created in each region where a SQL Server is created that will be shared by all servers in that region.
+    3. Provided resource group should pre-exist in all the subscriptions (in scope), otherwise remediation will fail. 
 
 #### Policy Definition
-[Security - Subscription - EnableMicrosoftDefenderForSQL](Security%20-%20Subscription%20-%20EnableMicrosoftDefenderForSQL.json)
+[Security - SQL Server - AuditSqlServerAuditingSettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerAuditingSettings.json)
+(Policy to audit the SQL Auditing on SQL Server.)
```

Comment:
This policy should be able to audit both generic sql server and sql servers part of synapse workspace.

Reply:
hi Tarun, as discussed, resolving the comment.